src/Nav.tsx

```diff
@@ -57,125 +151,62 @@ export default function Nav({
-          ["Traits", TraitsPng],
-          ["Spells", SpellsPng],
-          ["Status-Effects", StatusEffectsPng],
-        ].map(([text, icon]) => (
-          <ListItem
-            key={text}
-            component={MuiRouterLink}
-            to={`/${text.toLowerCase()}`}
-          >
-            <ListItemIcon aria-hidden="true">
-              <Tooltip
-                title={isNavOpen ? "" : text}
-                describeChild
-                arrow
-                placement="right"
-              >
-                <img src={icon} width={iconWidth} alt={text} />
-              </Tooltip>
-            </ListItemIcon>
-            {isNavOpen && <ListItemText primary={text} sx={itemTextStyle} />}
-          </ListItem>
-        ))}
-        {[
-          [
-            "Source Code",
-            "https://github.com/rovermicrover/siralim-ultimate-ui",
-            isDarkTheme ? GitDarkSvg : GitLightSvg,
-          ],
-          ["Author", "https://github.com/rovermicrover/", AuthorJpg],
-          [
-            "SU on Steam",
-            "https://store.steampowered.com/app/1289810/Siralim_Ultimate/",
-            SteamJpg,
-          ],
-        ].map(([text, url, img]) => (
-          <ListItem
-            key={text}
-            component={Link}
-            href={url}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <ListItemIcon aria-hidden="true">
-              <Tooltip title={isNavOpen ? "" : text} arrow placement="right">
-                <img src={img} width={iconWidth} alt={text} />
-              </Tooltip>
-            </ListItemIcon>
-            {isNavOpen && <ListItemText primary={text} sx={itemTextStyle} />}
-          </ListItem>
-        ))}
-
-        {[
-          [
-            "Source 1",
-            "https://docs.google.com/spreadsheets/d/1RYRvKTCLLJxXrZ_7OOjG8j98L_fjE5KNHtLG4wHn9Xw/edit#gid=0",
-          ],
-          [
-            "Source 2",
-            "https://docs.google.com/spreadsheets/d/1qvWwf1fNB5jN8bJ8dFGAVzC7scgDCoBO-hglwjTT4iY/edit#gid=0",
-          ],
-          [
-            "Source 3",
-            "https://docs.google.com/spreadsheets/d/1hlS4iNB6Uj-KVKzmFEygZkeTfX-U73B9R2lXdV3d5I8/edit#gid=0",
-          ],
-        ].map(([text, url]) => (
-          <ListItem
-            key={text}
-            component={Link}
-            href={url}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <ListItemIcon aria-hidden="true">
-              <Tooltip title={isNavOpen ? "" : text} arrow placement="right">
-                <SourceIcon sx={{ fontSize: `${iconWidth}px` }} />
-              </Tooltip>
-            </ListItemIcon>
-            {isNavOpen && <ListItemText primary={text} sx={itemTextStyle} />}
+      <nav>
+        <List
+          sx={{
+            whiteSpace: "nowrap",
+            overflowX: "hidden",
+          }}
+        >
+          {iconLinks.map((data) => {
```

Comment:
Its more "JSX" to do 

```javascript 
bar.map((data) => (<foobar>Hello World!</foobar))
``` 

instead of 

```javascript 
bar.map((data) => { return (<foobar>Hello World!</foobar) })
```